user.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+
+# DTO - Data Transfer Object
+
+@dataclass
+class Todo:
```

Comment:
The class name should be User I think
```suggestion
class User:
```